web/renderer/components/pageComponents/ConnectionsPage/ExistingConnections/Item.tsx

```diff
@@ -49,15 +69,18 @@ export default function Item({
             </Button.Link>
             <div className={css.typeAndName}>
               <DatabaseTypeLabel conn={conn} />
-              <Button.Link onClick={onAdd} className={css.name}>
+              <Button.Link
+                onClick={forElectron && conn.isLocalDolt ? handleSubmit : onAdd}
```

Comment:
Might be clearer to just do `onClick={onSubmit}` here and handle `isLocalDolt` in that function